src/services/promoted_attribute_definition_parser.ts

```diff
@@ -1,6 +1,15 @@
-function parse(value) {
+interface DefinitionObject {
+    isPromoted: boolean;
+    labelType: string;
+    multiplicity: string;
+    numberPrecision: number;
+    promotedAlias: string;
+    inverseRelation: string;
+}
```

Comment:
Looking between lines 15 and 41, it seems that these are all optional?
```suggestion
interface DefinitionObject {
    isPromoted?: boolean;
    labelType?: string;
    multiplicity?: string;
    numberPrecision?: number;
    promotedAlias?: string;
    inverseRelation?: string;
}
```